modules/test-utils/src/main/java/org/jboss/wsf/test/AppclientHelper.java

```diff
@@ -167,9 +167,20 @@ private static AppclientProcess newAppclientProcess(final String archive, final
          final ProcessBuilder pb = new ProcessBuilder().command(args);
          // always propagate IPv6 related properties
          final StringBuilder javaOptsValue = new StringBuilder();
-         String additionalJVMArgs = System.getProperty("additionalJvmArgs");
+
+         // wildfly9 security manage flag changed from -Djava.security.manager to -secmgr.
+         // Can't pass -secmgr arg through arquillian because it breaks arquillian's
+         // config of our tests.
+         // the -secmgr flag MUST be provided as an input arg to jboss-modules so it must
+         // come after the jboss-modules.jar ref.
+         String additionalJVMArgs = System.getProperty("additionalJvmArgs", "");
          if (additionalJVMArgs != null) {
-            javaOptsValue.append(additionalJVMArgs).append(" ");
+
+            if ("-Djava.security.manager".equals(additionalJVMArgs)) {
+               System.setProperty("SECMGR", "true");
```

Comment:
Is it safe to adjust system properties ? There should be no side effects for following testcases.

`-secmgr` could be prepend to `configArg` in this block

Reply:
I believe setting system property here is safe for 2 reasons.  One, here it is temporary; it only existing
in the executing process of this test.  Two, property SECMGR is specific to wildfly (see standalone.sh and related documentation).  The sys property is specific to enabling the security manager.  
Setting flag -secmgr will not work.  These scripts do not use the appclient-config file.

Reply:
ok, thanks for details